forge-gui/res/cardsfolder/upcoming/junkyard_genius.txt

```diff
@@ -4,7 +4,7 @@ Types:Creature Human Artificer
 PT:2/2
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a tapped Powerstone token. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
 SVar:TrigToken:DB$ Token | TokenTapped$ True | TokenScript$ c_a_powerstone
-A:AB$ PumpAll | Cost$ 1 B R Sac<1/Creature.Other;Artifact/another creature or artifact> | ValidCards$ Creature.YouCtrl+Other | NumAtt$ 1 | KW$ Menace & Haste | SpellDescription$ Until end of turn, other creatures you control get +1/+0 and gain menace and haste.
+A:AB$ PumpAll | Cost$ 1 B R Sac<1/Creature.Other;Artifact.Other/another creature or artifact> | ValidCards$ Creature.YouCtrl+Other | NumAtt$ 1 | KW$ Menace & Haste | SpellDescription$ Until end of turn, other creatures you control get +1/+0 and gain menace and haste.
```

Comment:
No, I think that's wrong
there was some card with a ruling for that afair
it would have to be worded "Sacrifice another creature or *an* artifact"